tests/test_entities.py

```diff
@@ -69,7 +70,7 @@ class InitTestCase(TestCase):
     def setUpClass(cls):
         """Set a server configuration at ``cls.cfg``."""
         cls.cfg = config.ServerConfig('http://example.com')
-        cls.cfg_610 = config.ServerConfig('http://example.com', version='6.1.0')
+        cls.cfg_610 = config.ServerConfig('http://example.com', version='6.8.0')
```

Comment:
I wonder why you have `cls.cfg_610` not 680 here. I further wonder why you cannot just remove all the 6.1 stuff. Satellite 6.1 was quite different, but we do not need that code in master.

Reply:
Just missed updating the name here.

As you'll see I have removed entire tests for some entities, but that was focused on which unit tests started failing, and whether or not they were still relevant. I'll take a second look at how relevant this test is and at a minimum update the name to be consistent.

Reply:
Some removals may take some input from component owners, or a whole lot of investigation by me - we've got entity mixin function overrides that per their docblocks are to workaround BZ's that were written and closed years ago.